tests/cli/ursula/test_stake_via_allocation_contract.py

```diff
@@ -464,7 +464,9 @@ def test_collect_rewards_integration(click_runner,
     blockchain_alice.selection_buffer = 1
 
     M, N = 1, 1
-    expiration = maya.now() + datetime.timedelta(days=3)
+    days = 3
+    now = testerchain.w3.eth.getBlock(block_identifier='latest').timestamp
+    expiration = maya.MayaDT(now).add(days=days-1)
```

Comment:
This whole manually subtracting/adding a day is not intuitive at all.

Reply:
How about `expiration = maya.now() - timedelta(days=1)`?

Reply:
how about to rename `days` to `days_to_pay` or to something similar?

Reply:
I think I need to explain it better what is core of these changes:
Before this PR duration of policy was equal to periods for which Alice pays. Also was `firstPartialReward`, kind of alternative way to pay for current period.
Now duration in seconds and not in periods. But payment still in periods. Sort of border between Alice and staker/worker regarding fees. Also duration includes current period by default (start timestamp is `block.timestamp`), so we get payment periods == duration periods + 1 (I mean duration periods as ceiling of `(endTimestamp - block.timestamp)/secondsPerPeriod`). Of course it's possible that  payment periods == duration periods in case when `block.timestamp` == first second of current period and end timestamp == last second of any period but this situation is not used in tests